php-fpm-healthcheck

```diff
@@ -54,13 +54,17 @@ command -v grep 1> /dev/null || { >&2 echo "Make sure grep is installed (i.e. ap
 get_fpm_status() {
     if test "$VERBOSE" = 1; then printf "Trying to connect to php-fpm via: %s%s\\n" "$1" "$SCRIPT_NAME"; fi;
     
-    # Since I cannot use pipefail I'll just split these in two commands
     FPM_STATUS=$(env -i REQUEST_METHOD="$REQUEST_METHOD" SCRIPT_NAME="$SCRIPT_NAME" SCRIPT_FILENAME="$SCRIPT_FILENAME" "$FCGI_CMD_PATH" -bind -connect "$1" 2> /dev/null)
+    RESPONSE_CODE=$(echo "$FPM_STATUS" | grep -o -E "^Status:\s*\d{3}" | sed 's/[^0-9]*//g')
     FPM_STATUS=$(echo "$FPM_STATUS" | tail +5)
 
     if test "$VERBOSE" = 1; then printf "php-fpm status output:\\n%s\\n" "$FPM_STATUS"; fi;
 
-    if test "$FPM_STATUS" = "File not found."; then
+    if ! test "$RESPONSE_CODE" -eq "$RESPONSE_CODE" 2> /dev/null; then
```

Comment:
What's this statement doing?

Reply:
Actually took me a quite long time to remember what this thing is doing. Here I checked for a integer `$RESPONSE_CODE`.

Reply:
Doing some research on the reason the response code was empty, I found the RFC stating that CGI would consider status 200 if status header is omitted. So I am removing that validation. https://tools.ietf.org/html/rfc3875#section-6.2.1